change/@splunk-otel-001c4603-3bb6-4d6f-8e53-fb10e4f5e929.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Make w3c(tracecontext, baggage) the default propagation mechanism",
+  "packageName": "@splunk/otel",
+  "email": "Rauno56@gmail.com",
```

Comment:
What's this change directory for with the json files?

Reply:
Was introduced in https://github.com/signalfx/splunk-otel-js/commit/527edb50b4a677832d868b5a4b54e8dce9f3099e. Check [CONTRIBUTING.md](https://github.com/signalfx/splunk-otel-js/blob/main/CONTRIBUTING.md#contributing-via-pull-requests).

Reply:
Ah, thanks